File: 2497 - Counting Cycles.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n;
        int i = 0;
        while ((n = scanner.nextInt()) != -1) {
            System.out.println("Experiment " + (++i) + ": " + (n / 2) + " full cycle(s)");
        }
        scanner.close();
    }
}

File: 2497 - Counting Cycles.cpp
#include<iostream>
using namespace std;
int main()
{
  int n;
  int i = 0;
  while(cin>>n && n!=-1)
  {
    cout<<"Experiment "<<++i<<": "<<n/2<<" full cycle(s)"<<endl;
  }
    return 0;
}
